src/NzbDrone.Core/MediaFiles/MediaInfo/VideoFileInfoReader.cs

```diff
@@ -174,6 +176,20 @@ private VideoStream GetPrimaryVideoStream(IMediaAnalysis mediaAnalysis)
             return mediaAnalysis.VideoStreams.FirstOrDefault(s => !codecFilter.Contains(s.CodecName)) ?? mediaAnalysis.PrimaryVideoStream;
         }
 
+        private AudioStream GetPrimaryAudioStream(IMediaAnalysis mediaAnalysis)
+        {
+            var audioStreams = mediaAnalysis.AudioStreams;
+
+            if (audioStreams.Count <= 1 || audioStreams.Any(s => s.BitRate == 0))
```

Comment:
Should the `Any` be `All` since the sorting should take care of picking the best stream.